.github/workflows/ci.yml

```
@@ -0,0 +1,34 @@
+        run: npm ci
+      
+      - name: Run linter
+        run: npm run lint
+      
+      - name: Check formatting
+        run: npm run format:check
+      
+      - name: Run tests
+        run: npm run test
+        continue-on-error: true # テストが未設定のため一時的にエラーを無視
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)